Symfony/CS/Fixer/Contrib/HeaderCommentFixer.php

```diff
@@ -91,7 +91,7 @@ private static function encloseTextInComment($header)
         foreach ($lines as $line) {
             $comment .= rtrim(' * '.$line)."\n";
         }
-        $comment .= " */\n";
+        $comment .= ' */';
```

Comment:
As I said..
that change was NOT made on on 1.7 branch: https://github.com/FriendsOfPHP/PHP-CS-Fixer/blob/1.7/Symfony/CS/Fixer/Contrib/HeaderCommentFixer.php#L94
it was made on master branch: https://github.com/FriendsOfPHP/PHP-CS-Fixer/blob/master/Symfony/CS/Fixer/Contrib/HeaderCommentFixer.php#L97